front/src/interfaces/company.interface.ts

```diff
@@ -1,5 +1,59 @@
+import { User } from './user.interface';
+
+export interface Opportunity {
```

Comment:
I would move this into opportunity.interface.ts for consistency

Reply:
I saw a separate task about adding opportunities in Linear, so I added this comment in the commit msg:

```
Opportunity is expected to be replace by actual opportunity in a separate PR.
```